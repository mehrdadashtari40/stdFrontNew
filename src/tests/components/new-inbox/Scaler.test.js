import React from 'react';
import { shallow } from 'enzyme';
import ScalerChart from '../../../views/new-inbox/components/scaler';

jest.mock('react-i18next', () => ({
	withTranslation: () => Component => {
		Component.defaultProps = { ...Component.defaultProps, t: () => '' };
		return Component;
	},
}));
/* Start of SnapShot Test */

test('Make Snapshot file , ScalerChart Component', () => {
	const wrapper = shallow(<ScalerChart title="test title" value="test value" unit="test unit" />);
	expect(wrapper).toMatchSnapshot();
});
