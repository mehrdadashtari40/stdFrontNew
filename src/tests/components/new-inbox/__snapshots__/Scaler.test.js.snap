// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make Snapshot file , ScalerChart Component 1`] = `
<div
  className="row"
>
  <div>
    <p>
      test title
    </p>
    <h4>
      test value
    </h4>
    <p>
      test unit
    </p>
  </div>
</div>
`;
